identity-infrastructure/src/main/java/life/qbic/identity/infrastructure/broadcasting/MessageDispatcher.java

```diff
@@ -0,0 +1,40 @@
+package life.qbic.identity.infrastructure.broadcasting;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Objects;
+import life.qbic.identity.application.communication.broadcasting.EventHub;
+import life.qbic.identity.application.communication.broadcasting.IntegrationEvent;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jms.core.JmsTemplate;
+import org.springframework.stereotype.Component;
+
+/**
+ * <b>Message Dispatcher</b>
+ * <p>
+ * Implementation of the {@link EventHub} interface, broadcasting events to the messaging
+ * middleware.
+ *
+ * @since 1.0.0
+ */
+@Component
+public class MessageDispatcher implements EventHub {
+
+  private static final String QUEUE = "User";
```

Comment:
What is the meaning/use of this String?

Reply:
Messages will be send to the queue/topic "User"